src/inverted-index.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+class InvertedIndex {
+
+  /**
+   * class constructor
+   * @constructor
+   */
+  constructor() {
+    this.indexedFiles = this.indexedFiles || [];
+  }
+
+
+  /**  strips all non-character values.
+   *
+   *  @param {String} text to be stripped
+   *  @return {Array} array of stripped values
+   */
+  strip(text) {
+    return text.toLowerCase()
+      .replace(/[^a-z0-9\s]/g, '')
+      .split(/\s+/g);
+  }
+
+
+  /** creates an inverted index
+   * @param {String} file name
+   * @return {Object} data to be indexed
+   */
+  createIndex(fileName, fileData) {
+    let terms = {};
+    let count = 0;
+    if (typeof (fileData) !== 'object') {
+      return 'invalid json file';
+    }
+   try{
```

Comment:
looping through objects with multiple levels will fail .. this try takes care of extra level of objects